go.mod

```diff
@@ -38,5 +38,4 @@ replace github.com/99designs/keyring => github.com/crypto-org-chain/keyring v1.1
 
 replace github.com/dgrijalva/jwt-go => github.com/dgrijalva/jwt-go/v4 v4.0.0-preview1
 
-// FIXME: back to upstream release once upgrade non-determinism is gone
-replace github.com/cosmos/cosmos-sdk => github.com/crypto-org-chain/cosmos-sdk v0.44.0-upgradefix
+replace github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.44.1
```

Comment:
is the replace still needed?

Reply:
ibc-go 1.2.0 depends on 0.44.0 -- without the replace, 0.44.0 will still be in go.sum... so just put it there to force 0.44.1